zstacklib/zstacklib/utils/pci.py

```
@@ -568,29 +568,399 @@ def get_vm_pci_device_address_by_host_address(vm_dom, host_address):
+    return (0, output, "")
+
+
+def collect_pci_devices_with_dependencies(pciDeviceAddress, sysfs_root=None):
+    """Find sibling PCI functions for a device.
+
+    When sysfs_root is provided, reads from sysfs instead of calling lspci.
+    """
+    if sysfs_root is None:
+        devices = []
+        base_address = pciDeviceAddress.rsplit('.', 1)[0]
+        r, o, e = bash_roe("lspci -s %s" % base_address)
```

Ruff (0.15.2)

[warning] Unpacked variable `e` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)